tasks/sep/mode-001-MCP-L1/VALIDATION.md

```diff
@@ -0,0 +1,340 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of the upgrade
+transaction.
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## State Overrides
+
+The following state overrides should be seen:
+
+### `0xE75Cd021F520B160BF6b54D472Fa15e52aFe5aDD` (The 1 of 1 `ProxyAdmin` owner Safe)
+
+Links:
+- [Etherscan](https://sepolia.etherscan.io/address/0xE75Cd021F520B160BF6b54D472Fa15e52aFe5aDD)
+
+Enables the simulation by reducing the threshold to 1:
```

Comment:
Good point, changed to "setting". This way it remains true even though it's a no-op on sepolia. Technically the override is not needed for sepolia, but it's easier to keep since this happens automatically as part of our sim library

We should also open a PR to change this phrasing in the metal file